src/openrct2/ride/VehiclePaint.cpp

```diff
@@ -3136,9 +3052,9 @@ static constexpr const vehicle_sprite_func PaintFunctionsByPitch[] = {
     VehiclePitchDiagDown12,
     VehiclePitchDiagDown25,
     VehiclePitchDiagDown60,
-    VehiclePitchInvertingDown60,
```

Comment:
I take it this was just a bunch of mistaken names?

Reply:
yes